src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -847,7 +847,7 @@ private static bool IsViableType(LookupResult result)
             {
                 return TypeWithAnnotations.Create(new ExtendedErrorTypeSymbol(
                     Compilation.Assembly.GlobalNamespace, identifierValueText, 0,
-                    new CSDiagnosticInfo(ErrorCode.ERR_SingleTypeNameNotFound)));
+                    new CSDiagnosticInfo(ErrorCode.ERR_SingleTypeNameNotFound, identifierValueText)));
```

Comment:
Which test failed this?

I think this change fixes https://github.com/dotnet/roslyn/issues/59417 but how can we observe that through a test?

Reply:
There were a number of tests that failed, including `IOperationTests_IConversionExpression.ConversionExpression_Implicit_ReferenceConversion_InvalidSyntax` and `IOperationTests_ISizeOfExpression.TestSizeOfExpression_MissingArgument`. The `DiagnosticInfo` is used in the resulting `ExtendedErrorTypeSymbol` but it's not clear how to construct a test where the `ExtendedErrorTypeSymbol.ErrorInfo` is reported from such cases though.